lms/djangoapps/instructor/views/api.py

```diff
@@ -10,14 +10,13 @@
 import logging
 import re
 import time
-import requests
```

Comment:
All of these unrelated changes (cleaning up imports and i18n) really ought to be split out into separate commits. Anyone that tries to decipher the logic of this commit in the future will be forced to wade through completely unrelated changes.

I wouldn't object to including them in the same PR, but there should be distinct, logical changesets:
- Commit 1: Display certificate generation history in reverse chronological order
- Commit 2: Add i18n to strings
- Commit 3: Remove unused imports

Not only is the initial review easier, but more importantly, it makes the code more decipherable (and subsequently more maintainable) by future developers.